.github/workflows/sync_issue-labels.yml

```diff
@@ -0,0 +1,37 @@
+name: Sync Issue Labels
+on:
+  issues:
+    types: [opened]
+jobs:
+  label-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: View context attributes
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const keywords = {
+              'techdocs*|tech-docs|tech docs': 'docs-like-code',
+              'search*': 'search',
+              'catalog*': 'catalog',
+              'scaffolder*': 'scaffolder',
```

Comment:
no need for the `*` in these right? Just ends up matching Scaffolderrrrrrrrr

Reply:
ah you're right 👍